azurerm/internal/services/frontdoor/frontdoor_resource.go

```diff
@@ -423,17 +427,21 @@ func resourceArmFrontDoor() *schema.Resource {
 							Default:  0,
 						},
 						"custom_https_provisioning_enabled": {
-							Type:     schema.TypeBool,
-							Required: true,
+							Type:       schema.TypeBool,
+							Optional:   true,
+							Computed:   true,
+							Deprecated: "Deprecated in favour of `custom_https_configuration` resource",
 						},
 						"web_application_firewall_policy_link_id": {
 							Type:     schema.TypeString,
 							Optional: true,
 						},
 						"custom_https_configuration": {
-							Type:     schema.TypeList,
-							Optional: true,
-							MaxItems: 1,
+							Type:       schema.TypeList,
+							Optional:   true,
+							Computed:   true,
+							MaxItems:   1,
+							Deprecated: "Deprecated in favour of `custom_https_configuration` resource",
```

Comment:
shouldn't it use the full name?
```suggestion
							Deprecated: "Deprecated in favour of `azurerm_frontdoor_custom_https_configuration` resource",
```

Looking at the other deprecated config it looks to be all changing attributes inside of the same resource which is why it's not qualified